lib/mayaUsd/fileio/utils/meshWriteUtils.h

```diff
@@ -29,11 +29,75 @@
 #include <pxr/usd/usdGeom/mesh.h>
 #include <pxr/usd/usdUtils/sparseValueWriter.h>
 
+#include <maya/MBoundingBox.h>
 #include <maya/MDagPath.h>
 #include <maya/MFnMesh.h>
 #include <maya/MObject.h>
 #include <maya/MString.h>
 
```

Comment:
The naming and namespacing is getting a bit muddied in here. The new functions are all in the global namespace with a `maya` prefix, while everything else is inside `PXR_NS`, then further scoped to `UsdMayaMeshWriteUtils`. Is there a reason the mesh-specific functions can't go in the `UsdMayaMeshWriteUtils` namespace? And it would be great to move the non-mesh specific functions out of this file.

Reply:
I was having trouble trying to decide where exactly to place my utlity functions. I'd be ok with moving the mesh-specific ones to that namespace, though generally I wasn't expecting to expose them to other translation units until they were actually needed.

Reply:
Yeah, namespacing in general is a little confusing and all over the place in general, the result of a few iterations of big code moves and merges. It'll take a focused clean-up pass to address a bunch of that later, but good to at least have these related functions grouped together for now, thanks!